### ISSUE TITLE
'High Contrast Black' theme Asset Upload Message color contrast

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1002]
**Describe the bug**
The 'Asset Upload' message text color contrast ration does not meet the minimum criteria i.e. `4.5:1`

> What version of the Fundamental Library Styles are you using?
v0.9.1-rc.0

> What front-end framework are you implementing in, e.g., Angular, React, etc.?
React

**To Reproduce**
1. Go to https://deploy-preview-928--fundamental-styles.netlify.app/components/asset-upload.html
2. Check the 'Upload Link' message color contrast with the background

**Expected behavior**
The 'Upload Link' message color contrast ratio should be 4.5:1 or above. Ideally 7:1 or above for WCAG AAA compliance.

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81739863-ea743400-9450-11ea-8443-aa6d62a13294.png)
![image](https://user-images.githubusercontent.com/43764832/81739353-0cb98200-9450-11ea-8632-af7eb65075d9.png)
![image](https://user-images.githubusercontent.com/43764832/81739945-0f68a700-9451-11ea-9e6e-e74dfd096130.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by Mike-Diaz** [comment]
Fixed as of version 0.13.0 - Contrast ratio now 13.7:1 - Component replaced with `File Uploader`
<img width="304" alt="Screen Shot 2020-10-07 at 6 23 26 PM" src="https://user-images.githubusercontent.com/21978807/95530488-d9453a00-0992-11eb-930f-f3c8f66fd608.png">
